NUnitConsole.sln

```diff
@@ -20,7 +20,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		src\Directory.Build.props = src\Directory.Build.props
 		GitReleaseManager.yaml = GitReleaseManager.yaml
 		GitVersion.yml = GitVersion.yml
-		global.json = global.json
```

Comment:
Suggest to restore this line so `global.json` can be viewed/edited from the solution.